src/app/hearing/services/hearing.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { AppConfig } from '../../app.config';
+import { HttpClient } from '@angular/common/http';
+import { Hearing } from '../models/hearing';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs/Observable';
+import { NotesPopulatorService } from '../../notes/services/notes-populator.service';
+
+@Injectable()
+export class HearingService {
+  constructor(
+    private readonly http: HttpClient,
+    private readonly config: AppConfig,
+    private readonly notesPopulatorService: NotesPopulatorService
+  ) {
+
+  }
+
+  getById(id: string): Observable<any> {
+    return this.http
+      .get<Hearing>(`${this.config.getApiUrl()}/hearing/${id}/with-sessions`)
+      .pipe(map(data => {
```

Comment:
you dont have to use `.pipe` while you are using just one inner function. You could use just `map`